assets/app/styles/_core.less

```diff
@@ -18,6 +19,9 @@ html, body {
     background-color: @body-bg;
     padding-bottom: @grid-gutter-width * 2;
     .flex(@columns: 1);
+    h1 {
+      word-wrap: break-word;
```

Comment:
Switched to word-wrap: break-word (), because text-overflow: ellipsis;  overflow: hidden caused type descenders to be hidden, due to Bootstrap's default heading line-height: 1.1 setting and changing this would have larger consequences.